src/porepy/models/constitutive_laws.py

```diff
@@ -3947,6 +3966,47 @@ def stiffness_tensor(self, subdomain: pp.Grid) -> pp.FourthOrderTensor:
         mu = self.solid.shear_modulus() * np.ones(subdomain.num_cells)
         return pp.FourthOrderTensor(mu, lmbda)
 
+    def characteristic_contact_traction(
+        self, subdomains: list[pp.Grid]
+    ) -> pp.ad.Operator:
+        """Characteristic traction [Pa].
+
+        The value is computed from the solid constants and the characteristic
+        displacement. Inversion of this relationship, i.e.,
+        u_char=u_char(t_char), is possible if the characteristic traction is
```

Comment:
I think I understand the reasoning behind this comment, but the implication (presumably on the client side of this function) is unclear to me. Writing a tutorial may be a good idea indeed.

Reply:
I tried to clarify a bit.